CuratedContent/SoftwareSustainabilityMatrix.md

```diff
@@ -0,0 +1,81 @@
+# Software Sustainability Matrix
+
+<!--deck text start-->
+The Software Sustainability Matrix (SSM) is an effort to quantify the sustainability of software packages with the goal of guiding efforts to improve the sustainability of software package ecosystems and individual software packages and highlight where to focus improvement efforts.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe "Roscoe A. Bartlett GitHub Profile")
+#### Publication date: ???
+
+Resource information | Details
+:--- | :--- 
+Article title |  [How Sustainable is Your Software?](https://www.kitware.com/how-sustainable-is-your-software/)
+Authors | Will Schroeder, Jess Tate, Jean-Christophe Fillion-Robin, Christopher Johnson, Dženan Zukić, Matt McCormick, Lee A. Newberg, Daniel White, Alexandra Warner, Ross Whitaker and Rob MacLeod
+Publication Details | [Kitware Blog](https://www.kitware.com/blog), January 3, 2020
+Focus | Software sustainability
+
+The Software Sustainability Matrix (SSM) is a joint effort of the Scientific Computing and Imaging (SCI) Institute and Kitware to improve the health of the open-source package ecosystem making up the foundational software portfolio for the Center for Integrative Biomedical Computing (CIBC) lead by the SCI for the National Institutes of Health (NIH) <sup>[1],[2]</sup>.
+The goal of the SSM effort is to create numerical scores for different aspects of software sustainability for a software package as well of a combined sustainability score.
+These SSM sustainability scores are being to guide efforts to improve the sustainability of the software package ecosystem making to CIBC software portfolio, with the hopes that it will encourage support and contributions from the open-source community and reduce the cost and risks to sustain this collection software.
```

Comment:
@bartlettroscoe I tried to make a suggestion to fix this sentence starting with changing "being" to "being used", but then stumbled on what you are saying by "making to CIBC software porfolio". I think you will need to fix it.

Reply:
@pagrubel, I reworded this sentence in commit e325da88e801036e9a5daf42009a94239c9f5437.  Please take a look.